transformations/casual2formal/README.md

```diff
@@ -0,0 +1,15 @@
+# Casual to Formal 🦎 → 🐍
+This transformation transfers text style from casual (informal) to formal.
+
+Author Name: Rishabh Gupta (rishabh19089@iiitd.ac.in)
+
+## What type of a transformation is this?
+This transformation acts like a paraphraser by performing style transfer. It changes the tone of the text from informal to formal.
+Example: What you upto? --> What are you currently doing? 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
```

Comment:
Please add what is the type of training data used here - I think the source of the model & code are taken from here: https://github.com/PrithivirajDamodaran/Styleformer
You should mention it here.

Reply:
Added it to README